### ISSUE TITLE
Further build issues with Poppler 0.71

### ISSUE THREAD

**Issue by joebonrichie** [OpenPrinting/cups-filters#75]
After applying 48625da I am still getting build issues with poppler 0.71

```
  CXX      pdftoopvp-OPVPSplashClip.o
filter/pdftoopvp/oprs/OPVPSplashClip.cxx: In member function ‘OPVPSplashPath* OPVPSplashClip::makePath()’:
filter/pdftoopvp/oprs/OPVPSplashClip.cxx:43:16: error: ‘class SplashXPathScanner’ has no member named ‘getNextSpan’; did you mean ‘testSpan’?
   scanners[0]->getNextSpan(tsyMin-2,&x0,&x1);
                ^~~~~~~~~~~
                testSpan
filter/pdftoopvp/oprs/OPVPSplashClip.cxx:50:25: error: ‘class SplashXPathScanner’ has no member named ‘getNextSpan’; did you mean ‘testSpan’?
     while (scanners[0]->getNextSpan(y,&x0,&x1)) {
                         ^~~~~~~~~~~
                         testSpan
filter/pdftoopvp/oprs/OPVPSplashClip.cxx:62:27: error: ‘class SplashXPathScanner’ has no member named ‘getNextSpan’; did you mean ‘testSpan’?
       while (scanners[j]->getNextSpan(y,&x0,&x1)) {
                           ^~~~~~~~~~~
                           testSpan
make[1]: *** [Makefile:4082: pdftoopvp-OPVPSplashClip.o] Error 1
```

After a small search I found a related change in poppler:-

https://cgit.freedesktop.org/poppler/poppler/commit/?id=68fdbfd0b159e8cf146c480f0d14f5d7adfd5806

Which seems to be causing the error. Modifying to match the new upstream behaviour should hopefully resolve the error.

**Comment by tillkamppeter** [comment]
Thank you for the hint. Most people will not take note of this failure as pdftoopvp is not built by default.
Unfortunately, the development branch of Ubuntu (Disco, 19.04) does not yet have Poppler 0.71. Could you make the patch, test it, and post it as a pull request here?
Another question: For what are you using pdftoopvp? Which printer? Which driver?

**Comment by tillkamppeter** [comment]
Please make the patch, as I even do not have the possibility to test building it.
Please also make it working with both the old and new Poppler, by using conditionals ("`#ifdef ...`").
Thanks in advance.

**Comment by StefanBruens** [comment]
See #83

**Comment by tillkamppeter** [comment]
Fixed with pull request #83.
